fiaas_deploy_daemon/deployer/kubernetes/service_account.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+# -*- coding: utf-8
+
+# Copyright 2017-2019 The FIAAS Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from __future__ import absolute_import
+
+import logging
+
+from k8s.client import NotFound
+from k8s.models.common import ObjectMeta, LocalObjectReference
+from k8s.models.service_account import ServiceAccount
+
+from fiaas_deploy_daemon.retry import retry_on_upsert_conflict
+from fiaas_deploy_daemon.tools import merge_dicts
+
+
+LOG = logging.getLogger(__name__)
+
+
+class ServiceAccountDeployer(object):
+    def __init__(self, config, owner_references):
+        self._owner_references = owner_references
+
+    def deploy(self, app_spec, labels):
+        self._create(app_spec, labels)
+
+    def delete(self, app_spec):
```

Comment:
As mentioned when we chatted, this method isn't necessary; since we're setting the ownerReferences, the deletion of dependent objects will happen automatically, and we don't need to worry whether or not we're deleting something that fiaas didn't create

Reply:
I will keep it anyways. WDYT?